Azure/sonic-buildimage
# [Mellanox] Update ASIC packages to: SAI 1.17, SDK 4.4.1306, FW xx.200…

…8.1310

Signed-off-by: Volodymyr Samotiy <volodymyrs@mellanox.com>



**- Why I did it**
Updated Mellanox ASIC packages to include below changes.

SAI
- Add switch type attr
- Fix ECMP max groups logic
- Add set issu log level for spc2/spc3, as now issu is supported
- Set vlan max swid = 0 on sdk init, as only single swid is needed, for efficient resource usage
- Add sflow throguh netlink channel
- Add RX/TX oversize pkts port stats

SDK/FW
- Added support for Finisar 100GbE SWDM Transceiver FTLC9152RGPL.
- Spectrum-2 Added support for 10G BaseT modules
- Added link LED support for SN4600C.
- Counters | In SDK debug dump, the incorrect counter type appears for vtraps.
- WJH | Without any traffic or events on the idle system, the CPU load is constantly above 4%
- WJH | WJH filter currently cannot filter by PORT for buffer drop reason.
- Spectrum | ACL, Unbind, Lazy Delete | Running Lazy Delete together with auto_unbind may cause rate condition errors. To work work with Lazy Delete use new INIT parameter "acl_manual_unbind" so that ACLs will notbe removed automatically when binding point is deleted.
- Spectrum | ISSU | In ISSU mode, when querying for the number of configurable buffers, using the API sx_api_cos_port_buff_type_get with the count parameter as 0, the API returns the number for NORMAL mode instead.
- Spectrum-2 | BER | BER monitor counts raw errors instead of effective errors
- Spectrum-2 | BER | Connecting to ConnectX-5 adapter card with copper splitter cable MCP7H50-V001R30 in 1
- Spectrum-2 | Cables | Link flaps in 200GbE with AOM Optic cable MMA1T00-VS
- Spectrum-3 | Speeds, Link | When moving from a 400GbE link to a 1GbE link, packets may drop for 1msec right after link up
- Spectrum-3 | Cables, Speeds | Using 400GbE with 3rd party systems is not supported
- Spectrum-3 | LAG | After a while, LAG members become out of sync with one another
- Spectrum-3 | VLAN, Ports | Packets with VLAN headers are sent to
- When editing a MAC that points to a new ECMP witch a next-hop that points to  another ECMP SDK  operations never ends
- Spectrum-3 | In rare cases, creating a new RIF fails and FW doesn't finish the operation
- Spectrum | The maximum offset in neighbour (SYNC) timestamps is out of default range on Slave node
- Spectrum | In some case Packet timestamp is need been match to the arrived packets and only match when another packet arrives 
- Momentary traffic loss in warm boot when PG-0 is in used
- Telemetry threshold events stop to arrive after LAG port member change
- Spectrum-3 | when connected to a Spectrum-3 or Spectrum-2 switch after port up/down, link down counter increases twice instead of once 
- Spectrum-2 | 40G AOC optical cable becomes unusable and locked up

**- How I did it**
Updated versions and submodule pointers.

**- How to verify it**
Build an image and verify that all is up and running after deploying to the switch.

**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**

[Mellanox] Update ASIC packages to: SAI 1.17, SDK 4.4.1306, FW xx.2008.1310
